src/iconv.cpp

```diff
@@ -110,7 +110,7 @@ IConv::convert(const std::string& text)
 
     // We try to avoid to much copying around, so we write directly into
     // a std::string
-    tinygettext_ICONV_CONST char* inbuf = const_cast<char*>(&text[0]);
+    const char* inbuf = const_cast<char*>(&text[0]);
```

Comment:
I will look that over.  I am using the win32 gnu version of ICONV, but I think the combination of that and SDL messed things up.  That said, what is the issue with it always being a const?